src/pl/michal/interfejs/MySQLaccess.java

```diff
@@ -0,0 +1,202 @@
+package pl.michal.interfejs;
+
+import java.io.FileInputStream;
+import java.sql.*;
+import java.util.Date;
+import java.util.Properties;
+
+public class MySQLaccess {
+    private Connection connection = null;
+    private Statement statement = null;
+    private PreparedStatement preparedStatement = null;
+    private ResultSet resultSet = null;
+
+    public MySQLaccess() {
+        try {
+            this.CreateConnection("db.properties");
```

Comment:
1. 'this' jest tutaj zbędne, wiadomo że metoda działa na tym obiekcie, 'this' przydaje się w konstruktorze jedynie wtedy, gdy argument konstruktora ma taką samą nazwę jak nazwa membera który chcesz ustawić, np
`MySQLaccess(String name){
this.name = name;
}`
Druga rzecz to nazwa metody CreateConnection - to razi. Zmień na createConnection - pozostałe również. Później bez IDE ciężko się zorientować kiedy mamy wywołanie metody, kiedy tworzenie obiektu, itp.